### DISCUSSION TITLE
blog/mikro-orm-6-released

### DISCUSSION THREAD

**Discussion by giscus** [mikro-orm/mikro-orm#5080] [Comments]
# blog/mikro-orm-6-released

After more than a year in the development, I am thrilled to announce the next major version of MikroORM has just become stable. It brings many improvements throughout the whole system, and doubles down on type-safety and strictness.

https://mikro-orm.io/blog/mikro-orm-6-released

**Comment by eddienubes** [comment]
Wow, congratulations guys!

**Comment by rubiin** [comment]
Damn <3 much awaited

**Comment by lineCode** [comment]
wow

**Comment by Untaek** [comment]
Nice work!!
